tests/scaffold/basic_vm_skip.t.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.0;
+
+import {Test} from "forge-std/Test.sol";
+
+contract HashPairTestSanitize is Test {
+    function test_ShouldNeverRevert() external {
+        vm.skip(true);
+        // It should never revert.
```

Comment:
I did it mostly because:
- I personally find tidier to keep the original "It" alongside the relevant assertion (for instance https://github.com/Bananapus/nana-swap-terminal/blob/f0c3f6da57e7b4039cc5bd2a29ad68d958b4a985/test/Units/AddDefaultPool.t.sol#L35) and couldn't find other way of doing in Sablier (for instance - iic, prb sometimes used the "it" as dev natspec, but most of the time removed it altogether). How do you usually use it? 
- It felt easier for extra-statements (especially bound), to keep them at the top, like the cool magic fixture that they are

No strongly opinionated neither, except for performance uber alles:)

Reply:
even after removing the vm.skip (great point, I kinda forgot it was a temp stuff), the other ones should, imo, remain on top -> it looked then easier to add all statement together in the hir translator (but more insert() tho)

```Rust
test_ShouldNeverRevert(uint256 fuzzed) external {
    fuzzed = bound(...);
    vm.startPrank(alice);

    // It should never revert.
    assert_eq(...)
}
```
and something like 
```Rust
// Add all statements at the start of the function.
match self.cfg.StatementToAdd {
    StatementToAdd { with_skip: true, _, _ } => 
        actions.insert(
            0,
            Hir::Statement(hir::Statement {
                ty: hir::StatementType::VmSkip,
            }),
        ),
    StatementToAdd {_, with_bound: true, _ } => 
        actions.insert(
            0,
            Hir::Statement(hir::Statement {
                ty: hir::StatementType::Bound,
            }),
        ),

(...)

  }
```

Reply:
Hmm, but then, it's maybe even better to push them before the `it` 🤔

Reply:
Hehehe, yeah, I think so too. Also, note it may have multiple `it`s and some action descriptions (see https://github.com/alexfertel/bulloak/issues/29)

In any case, I'm fine with both things, so feel free to choose the one you deem more reasonable.